doc/contributing/documentation_guide.md

```diff
@@ -137,6 +137,15 @@ or [list](rdoc-ref:RDoc::Markup@Simple+Lists)
 should be preceded by and followed by a blank line.
 This is unnecessary for the HTML output, but helps in the `ri` output.
 
+### \Method Names
+
+For a method name in text:
+
+- When the class name is included, use a dot or a hash mark:
```

Comment:
I'm a bit confused about what what you mean by "When the class name is included". I think it would make more sense to swap the order of these two bullet points, the `::bar` and `#baz` syntax can only be used to refer to methods in the current class or module while `Foo.bar` and `Foo#baz` syntax is required in all other cases.